-------------------------------------------------------------------------------
Test set: io.cloudboost.CloudQueryTest
-------------------------------------------------------------------------------
Tests run: 29, Failures: 10, Errors: 19, Skipped: 0, Time elapsed: 0.434 sec <<< FAILURE!
selectColumn(io.cloudboost.CloudQueryTest)  Time elapsed: 0.024 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$1.done(CloudQueryTest.java:25)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.selectColumn(CloudQueryTest.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
lessThanEqualTo(io.cloudboost.CloudQueryTest)  Time elapsed: 0.012 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.lessThanEqualTo(CloudQuery.java:348)
	at io.cloudboost.CloudQueryTest$13.done(CloudQueryTest.java:371)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.lessThanEqualTo(CloudQueryTest.java:367)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
ascendingOrder(io.cloudboost.CloudQueryTest)  Time elapsed: 0.02 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$15$1.done(CloudQueryTest.java:440)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudQueryTest$15.done(CloudQueryTest.java:437)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.ascendingOrder(CloudQueryTest.java:432)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
descendingOrder(io.cloudboost.CloudQueryTest)  Time elapsed: 0.019 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$16$1.done(CloudQueryTest.java:470)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudQueryTest$16.done(CloudQueryTest.java:467)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.descendingOrder(CloudQueryTest.java:462)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
findById(io.cloudboost.CloudQueryTest)  Time elapsed: 0.016 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$17.done(CloudQueryTest.java:497)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.findById(CloudQueryTest.java:492)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
doesNotExists(io.cloudboost.CloudQueryTest)  Time elapsed: 0.015 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.doesNotExists(CloudQuery.java:168)
	at io.cloudboost.CloudQueryTest$19.done(CloudQueryTest.java:551)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.doesNotExists(CloudQueryTest.java:547)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
getElementWithRelation(io.cloudboost.CloudQueryTest)  Time elapsed: 0.023 sec  <<< ERROR!
org.json.JSONException: JSONObject["_id"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.notEqualTo(CloudQuery.java:122)
	at io.cloudboost.CloudQueryTest$20$1.done(CloudQueryTest.java:584)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest$20.done(CloudQueryTest.java:580)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.getElementWithRelation(CloudQueryTest.java:575)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
saveRelation(io.cloudboost.CloudQueryTest)  Time elapsed: 0.009 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$21.done(CloudQueryTest.java:621)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.saveRelation(CloudQueryTest.java:617)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
saveMultiJoin(io.cloudboost.CloudQueryTest)  Time elapsed: 0.016 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$22.done(CloudQueryTest.java:645)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.saveMultiJoin(CloudQueryTest.java:641)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
includeRelationlObjectInQuery(io.cloudboost.CloudQueryTest)  Time elapsed: 0.018 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$23.done(CloudQueryTest.java:669)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.includeRelationlObjectInQuery(CloudQueryTest.java:665)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
includeRelationObjectOnDistinct(io.cloudboost.CloudQueryTest)  Time elapsed: 0.013 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$24.done(CloudQueryTest.java:711)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.includeRelationObjectOnDistinct(CloudQueryTest.java:707)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
queryOverLinkedColumnWithEqualTo(io.cloudboost.CloudQueryTest)  Time elapsed: 0.011 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$25.done(CloudQueryTest.java:749)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.queryOverLinkedColumnWithEqualTo(CloudQueryTest.java:745)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
containedInOverListOfCloudObject(io.cloudboost.CloudQueryTest)  Time elapsed: 0.012 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$26.done(CloudQueryTest.java:783)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.containedInOverListOfCloudObject(CloudQueryTest.java:779)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
includeFindById(io.cloudboost.CloudQueryTest)  Time elapsed: 0.01 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$27.done(CloudQueryTest.java:818)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.includeFindById(CloudQueryTest.java:814)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
getEncryptedPassword(io.cloudboost.CloudQueryTest)  Time elapsed: 0.036 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$28$1.done(CloudQueryTest.java:857)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudQueryTest$28.done(CloudQueryTest.java:853)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.getEncryptedPassword(CloudQueryTest.java:847)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
getEncryptedPasswordOverQuery(io.cloudboost.CloudQueryTest)  Time elapsed: 0.022 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$29.done(CloudQueryTest.java:881)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.getEncryptedPasswordOverQuery(CloudQueryTest.java:878)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
greaterThan(io.cloudboost.CloudQueryTest)  Time elapsed: 0.012 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.greaterThan(CloudQuery.java:278)
	at io.cloudboost.CloudQueryTest$10.done(CloudQueryTest.java:287)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.greaterThan(CloudQueryTest.java:283)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
greaterThanEqualTo(io.cloudboost.CloudQueryTest)  Time elapsed: 0.032 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.greaterThanEqualTo(CloudQuery.java:301)
	at io.cloudboost.CloudQueryTest$11.done(CloudQueryTest.java:315)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.greaterThanEqualTo(CloudQueryTest.java:311)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
lessThan(io.cloudboost.CloudQueryTest)  Time elapsed: 0.011 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.lessThan(CloudQuery.java:324)
	at io.cloudboost.CloudQueryTest$12.done(CloudQueryTest.java:343)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.lessThan(CloudQueryTest.java:339)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
selectColumnDistinct(io.cloudboost.CloudQueryTest)  Time elapsed: 0.009 sec  <<< ERROR!
java.lang.NullPointerException
	at io.cloudboost.CloudQueryTest$2.done(CloudQueryTest.java:58)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.selectColumnDistinct(CloudQueryTest.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
notEqualToWithNull(io.cloudboost.CloudQueryTest)  Time elapsed: 0.014 sec  <<< ERROR!
org.json.JSONException: JSONObject["name"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.exists(CloudQuery.java:145)
	at io.cloudboost.CloudQuery.notEqualTo(CloudQuery.java:129)
	at io.cloudboost.CloudQueryTest$4.done(CloudQueryTest.java:121)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.notEqualToWithNull(CloudQueryTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
findOne(io.cloudboost.CloudQueryTest)  Time elapsed: 0.009 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$5.done(CloudQueryTest.java:154)
	at io.cloudboost.CloudQuery.findOne(CloudQuery.java:1054)
	at io.cloudboost.CloudQueryTest.findOne(CloudQueryTest.java:150)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
equalToWithNull(io.cloudboost.CloudQueryTest)  Time elapsed: 0.012 sec  <<< ERROR!
org.json.JSONException: JSONObject["name"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.doesNotExists(CloudQuery.java:168)
	at io.cloudboost.CloudQuery.equalTo(CloudQuery.java:101)
	at io.cloudboost.CloudQueryTest$3.done(CloudQueryTest.java:90)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.equalToWithNull(CloudQueryTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
findData(io.cloudboost.CloudQueryTest)  Time elapsed: 0.013 sec  <<< FAILURE!
junit.framework.AssertionFailedError: java.util.concurrent.ExecutionException: java.net.ConnectException: Connection refused: localhost/127.0.0.1:4730
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudQueryTest$6.done(CloudQueryTest.java:175)
	at io.cloudboost.CloudQuery.find(CloudQuery.java:963)
	at io.cloudboost.CloudQueryTest.findData(CloudQueryTest.java:171)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
notContianedIn(io.cloudboost.CloudQueryTest)  Time elapsed: 0.012 sec  <<< ERROR!
org.json.JSONException: JSONObject["subject"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getJSONObject(JSONObject.java:573)
	at io.cloudboost.CloudQuery.notContainedIn(CloudQuery.java:580)
	at io.cloudboost.CloudQueryTest$9.done(CloudQueryTest.java:253)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.notContianedIn(CloudQueryTest.java:249)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
startsWith(io.cloudboost.CloudQueryTest)  Time elapsed: 0.004 sec  <<< ERROR!
org.json.JSONException: JSONObject["name"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.startsWith(CloudQuery.java:762)
	at io.cloudboost.CloudQueryTest.startsWith(CloudQueryTest.java:225)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
exists(io.cloudboost.CloudQueryTest)  Time elapsed: 0.009 sec  <<< ERROR!
org.json.JSONException: JSONObject["age"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getString(JSONObject.java:654)
	at io.cloudboost.CloudQuery.exists(CloudQuery.java:145)
	at io.cloudboost.CloudQueryTest$18.done(CloudQueryTest.java:523)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.exists(CloudQueryTest.java:519)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
containsAll(io.cloudboost.CloudQueryTest)  Time elapsed: 0.011 sec  <<< ERROR!
org.json.JSONException: JSONObject["subject"] not found.
	at org.json.JSONObject.get(JSONObject.java:473)
	at org.json.JSONObject.getJSONObject(JSONObject.java:573)
	at io.cloudboost.CloudQuery.containsAll(CloudQuery.java:688)
	at io.cloudboost.CloudQueryTest$7.done(CloudQueryTest.java:198)
	at io.cloudboost.CloudObject.save(CloudObject.java:530)
	at io.cloudboost.CloudQueryTest.containsAll(CloudQueryTest.java:194)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
or(io.cloudboost.CloudQueryTest)  Time elapsed: 0.003 sec  <<< ERROR!
java.lang.ClassCastException: org.json.JSONArray cannot be cast to java.util.List
	at io.cloudboost.CloudQuery.or(CloudQuery.java:71)
	at io.cloudboost.CloudQueryTest.or(CloudQueryTest.java:400)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
